packages/web-frontend/src/utils/measures.js

```diff
@@ -271,37 +271,45 @@ export function getValueInfo(value, valueMapping, hiddenValues = {}) {
 export function getFormattedInfo(orgUnitData, measureOption) {
   const { key, valueMapping, type, displayedValueKey, scaleType, valueType } = measureOption;
 
+  const value = orgUnitData[key];
+  const valueInfo = getValueInfo(value, valueMapping);
+
   if (
     displayedValueKey &&
     (orgUnitData[displayedValueKey] || orgUnitData[displayedValueKey] === 0)
   ) {
     return {
-      value: formatDataValue(orgUnitData[displayedValueKey], valueType, orgUnitData.metadata),
+      formattedValue: formatDataValue(
+        orgUnitData[displayedValueKey],
+        valueType,
+        orgUnitData.metadata,
+      ),
+      valueInfo,
```

Comment:
Added `valueInfo` to the return object for this function as it's needed by the `PopupMarker` component to conditionally render the value for the measure.